docs/workshops/intro-to-accessibility.md

```diff
@@ -72,8 +69,392 @@ Try using your phone with the screen reader function.
+
+---
+
+### Different aspects of accessibility
+
+What is digital accessibility?
+
+"Digital accessibility is the ability of a website, mobile application or electronic document to be easily navigated and understood by a wide range of users, including those users who have visual, auditory, motor or cognitive impairments."
+
+**In the UK, 1 in 5 people are disabled**
+
+The concept of accessibility doesn’t just apply to disabled people – all users will have different needs at different times and in different circumstances.
+
+**Making things more accessible makes things better for all users.**
+
+_"My needs are different from yours and my needs today might be different tomorrow”_ - Charles Hall
+
+#### Vision impairments
+
+In the UK: almost 2 million people living with sight loss.
+360,000 are registered as blind or partially sighted.
+
+**Blindness**: there is a legal definition which doesn’t imply seeing nothing at all. Many can read from a screen, but can’t use a mouse.
+
+They often can’t see images, graphics or videos.
+
+If the layout of your page, the navigation or labels of your fields are not clear, they won’t be able to use your website or app.
+
+The main screen readers are:
+
+- NVDA - it’s free
+- JAWS - need a licence but good and have been here for a long time
+- Voice Over
+
+2 minutes video of a user (Kristy Viers) using an iPhone
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/l0xBsR5mIYs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### People with low vision
+
+- Might need to magnify and miss some information if you don’t **align your content on the left** or don’t group related items together.
+- Need to be able to scroll both vertically and horizontally
+- Can get lost in your page
+- Might need to change colour contrast on their device or browser
+
+![poster showing do and dont for screen readers](assets/a11y-part1/visually-impaired-screenreader.png)
+![poster showing do and don't for low vision](assets/a11y-part1/visually-impaired-low-vision.png)
+[Posters part of a set available here](https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf)
+
+#### Colour blindness
+
+People with colour vision deficiency find it difficult to identify and distinguish between certain colours.
+
+Total colour blindness (an inability to see any colour) is very rare.
+
+There are various types:
+
+- Deuteranomaly: reduce sensitivity to green
+- Protanopia: reduce sensitivity to red
+- Tritanopia: reduce sensitivity to blue
+
+For this people, information via colour alone won’t be understood or noticed.
+
+For example, an error message in red with no other marks that something is wrong.
+
+![4 images of tomatoes, normal vision, protanopia, tritanopia and deuteranomaly](assets/a11y-part1/different-types-color-blindness-photos-76.jpg)
+
+> **Exercise 1**:
```

Comment:
It'd be nice if you could add the amount of time you expect the exercise to take to each of these

Reply:
yes I'll add it after if you want , I have it in the slides or do you want me to add this to this actual pull request ?